haystack/preview/utils/set_seeds.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
Not sure this is something useful outside tests, can we move it away from the public API, somewhere in the `test` package?

Reply:
I believe this might make sense for benchmarks and evaluation scripts maybe?

Reply:
Also for training (if we decide to implement it in future)